Update notices documentation to reflect changes to Snackbar Notices (#4455)
woocommerce/woocommerce-gutenberg-products-block: docs/block-client-apis/notices.md

## StoreNoticesProvider

> The `StoreNoticesProvider` allows managing notices in the frontend. Notices are rendered on top of the children.

Reviewer: ```suggestion
The `StoreNoticesProvider` allows managing notices in the frontend. Notices are rendered before React Children.
```
